bin/methods/start.js

```diff
@@ -26,14 +30,18 @@ function start (flags, plugins = []) {
   })
 
   // Set defaults
-  if (!config.loader) config.loader = path.join(__dirname, '..', 'loader')
+  if (!config.loader) config.loader = path.join(__dirname, '../../node_modules/cordlr-loader')
   if (plugins.length) {
     if (!config.plugins) config.plugins = []
     config.plugins.push(...plugins)
   }
 
-  // Create bot.
-  const bot = cordlr(config)
+  // Create bot
+  const bot = new Client()
   bot.on('error', log.err())
```

Comment:
Maybe we should leave the error handling to the loader. Also, logging the error doesn't do much since the error is automatically logged if no listeners are attached to the event.